app/js/components/common/savedResultsTable.js

```diff
@@ -28,11 +29,13 @@ const SavedResultsTable = ({
           </thead>
           <tbody>
             {
-              results.map((result, index) => {
-                const { uuid, description } = result;
-                return (
+              results.map((result) => {
+                if (result.name != 'appointmentschedulingui.appointmentDataSetDefinition.dailyAppointments') {
```

Comment:
By the way, which exact rest url returns this? Are you able to reproduce it from here? https://modules-refapp.openmrs.org/openmrs/module/webservices/rest/apiDocs.htm